pages/community/overview.mdx

```diff
@@ -12,6 +12,7 @@ canonical: "/community/overview"
 - [Twitter](https://twitter.com/rescriptlang)
 - [Forum](https://forum.rescript-lang.org)
 - [ReScript GitHub Org](https://github.com/rescript-lang/)
+- [ReScript Online Meetup YouTube Channel](https://www.youtube.com/@ReScriptOnlineMeetup)
```

Comment:
Should we also add the link to the guild?
https://guild.host/rescript-online-meetup

Reply:
Done!